packages/conventional-changelog-angular/writer-opts.js

```diff
@@ -79,7 +79,13 @@ function getWriterOpts () {
         }
         if (context.host) {
           // User URLs.
-          commit.subject = commit.subject.replace(/\B@([a-z0-9](?:-?[a-z0-9]){0,38})/g, `[@$1](${context.host}/$1)`)
+          commit.subject = commit.subject.replace(/\B@([a-z0-9](?:-?[a-z0-9/]){0,38})/g, (_, username) => {
+            if (username.includes('/')) {
+              return `@${username}`
```

Comment:
I'm wondering if it'd be good if we also link the project. But it won't be generic enough (wouldn't work if its not a scoped module).

Reply:
That's really to hard to make it work for a not scoped module.

There is a _solution_, but it's not bullet-proof. 
On some of my projects, I use [Dependabot](https://dependabot.com/) which updates my dependencies automatically by doing PR like this:

![selection_340](https://user-images.githubusercontent.com/2103975/49923604-547f5700-feb4-11e8-982c-06e416b9b889.png)

Here, if we can detect `bump PACKAGE_NAME from X to Y`, then we can link the project even if it's not a scoped module. 
A more generic solution is to detect only `PACKAGE_NAME from X to Y`.

An other solution would be to detect all valid packages name in the commit subject, and then check for `https://registry.npmjs.org/:packageName`. But that means that those packages will be matched too: 
- https://registry.npmjs.org/bump
- https://registry.npmjs.org/from
- https://registry.npmjs.org/to

So I don't think we can make this... :disappointed:

Reply:
Yeah, not a good idea :)